password-reset-component.html

```diff
@@ -0,0 +1,62 @@
+<form name="regForm" class="form-horizontal" ng-submit="$ctrl.submit()">
+  <fieldset>
+    <br-input br-model="$ctrl.sysPassword"
+      br-options="{
+        name: 'sysPassword', type: 'password',
+        label: 'Password', disabled: {{$ctrl.loading}},
+        showValidation: true,
+        columns: {
+          label: 'col-md-3',
+          input: 'col-md-7',
+          help: 'col-md-offset-3 col-md-7'
+        }
+      }" ng-minlength="8" ng-maxlength="32" required>
+      <br-input-help>
+        Please enter a secure password that is between 8 and 32 characters long. The best passwords are
+        lengthy, memorable phrases like: the<strong>lemurs</strong>ride<strong>on</strong>the<strong>fortnight</strong>
+      </br-input-help>
+      <br-input-validation-errors>
+        <p ng-show="regForm.sysPassword.$error.required">
+          Password is required.
+        </p>
+        <p ng-show="regForm.sysPassword.$error.minlength">
+          Password is required to be at least 8 characters.
+        </p>
+        <p ng-show="regForm.sysPassword.$error.maxlength">
+          Password must not be longer than 32 characters.
+        </p>
+      </br-input-validation-errors>
+    </br-input>
+
+    <br-input br-model="$ctrl.passwordConfirmation"
+      br-options="{
+        name: 'passwordConfirmation', type: 'password',
+        label: 'Confirm Password', disabled: {{$ctrl.loading}},
+        showValidation: true,
+        columns: {
+          label: 'col-md-3',
+          input: 'col-md-7',
+          help: 'col-md-offset-3 col-md-7'
+        }
+      }" br-input-br-password-confirmation="$ctrl.sysPassword">
+      <br-input-help>
+        Please confirm your password.
+      </br-input-help>
+      <br-input-validation-errors>
+        <p ng-show="regForm.passwordConfirmation.$error.inputMatch">
+          Password and Confirm Password must match.
+        </p>
+      </br-input-validation-errors>
+    </br-input>
+  </fieldset>
+  <div class="form-group">
+    <div class="col-md-offset-3">
+      <button type="submit"
+        ng-disabled="$ctrl.loading || regForm.$invalid"
+        class="btn btn-primary btn-lg">Reset Password</button>
+      <span ng-show="$ctrl.loading">
+        <i class="fa fa-refresh fa-spin fa-lg"></i>
+      </span>
```

Comment:
Use the style above for activity spinner: https://github.com/digitalbazaar/bedrock-angular-authn-password/pull/1/files#r64653184